src/browser/tokenRefresh.ts

```
@@ -41,14 +44,71 @@ export async function refreshAccessToken(
+      this.refreshPromise = refreshAccessToken(this._refreshToken!)
+        .then((result) => {
+          if (result && 'accessToken' in result) {
+            this.token = result.accessToken;
+          } else if (result && result.error === 'permanent') {
+            this.permanentlyFailed = true;
+            apiLogger.debug(
+              '[token-refresh] Permanent failure, skipping future refreshes',
+            );
+            return null;
+          } else if (result && result.error === 'transient') {
+            apiLogger.warn('[token-refresh] Transient failure, proceeding with expiring token');
```

ESLint

[error] Replace `'[token-refresh]·Transient·failure,·proceeding·with·expiring·token'` with `⏎··············'[token-refresh]·Transient·failure,·proceeding·with·expiring·token',⏎············`

(prettier/prettier)